### PR TITLE
Taras vasylychyn

### PR DESCRIPTION
**Pull request by pettrichor**

### CODE REVIEW

File: Taras_Vasylychyn/lab_2/lab-2.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Львівський Національний Університет Природокористування
+Кафедра Інформаційних систем та Технологій
+
+Звіт про виконання лабораторної роботи №2
```
Comment (tatomyr) [inline Taras_Vasylychyn/lab_2/lab-2.md]:
```suggestion
# Звіт 
#### про виконання лабораторної роботи №2
```

File: Taras_Vasylychyn/lab_2/lab-2.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+"Variables and Types, Lists, Basic Operators, String Formatting, Basic String Operations, Dictionaries"
```
Comment (tatomyr) [inline Taras_Vasylychyn/lab_2/lab-2.md]:
```suggestion
### "Variables and Types, Lists, Basic Operators, String Formatting, Basic String Operations, Dictionaries"
```

File: Taras_Vasylychyn/README.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
 # Зміст
 
-[Практична робота №1](./lab_1/lab-1.py)
\ No newline at end of file
+[Практична робота №1](./lab_1/lab-1.py)
+[Практична робота №2](./lab_2/lab_2.py)
+[Практична робота №3](./lab_3/lab_3.py)
```
Comment (tatomyr) [inline Taras_Vasylychyn/README.md]:
```suggestion
[Практична робота №1](./lab_1/lab-1.md)
[Практична робота №2](./lab_2/lab-2.md)
[Практична робота №3](./lab_3/lab_3.md)
```

File: Taras_Vasylychyn/lab_1/lab-1.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+## Львівський Національний Університет Природокористування
+## Кафедра Інформаційних систем та Технологій
+
+
+
+### Звіт про виконання лабораторної роботи №1
```
Comment (tatomyr) [inline Taras_Vasylychyn/lab_1/lab-1.md:6]:
```suggestion
# Звіт про виконання лабораторної роботи №1
```

File: Taras_Vasylychyn/lab_1/lab-1.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+### "Ознайомлення з Python, GitHub"
```
Comment (tatomyr) [inline Taras_Vasylychyn/lab_1/lab-1.md]:
```suggestion
### Звіт про виконання лабораторної роботи №1
# "Ознайомлення з Python, GitHub"
```

File: Taras_Vasylychyn/lab_2/lab-2.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
+## Львівський Національний Університет Природокористування
+## Кафедра Інформаційних систем та Технологій
+
+
+
+### Звіт про виконання лабораторної роботи №1
+### "Ознайомлення з Python, GitHub"
```
Comment (tatomyr) [inline Taras_Vasylychyn/lab_2/lab-2.md]: A wrong theme perhaps?

File: Taras_Vasylychyn/lab_2/lab-2.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
+## Львівський Національний Університет Природокористування
+## Кафедра Інформаційних систем та Технологій
+
+
+
+### Звіт про виконання лабораторної роботи №1
```
Comment (tatomyr) [inline Taras_Vasylychyn/lab_2/lab-2.md]: 2?

File: Taras_Vasylychyn/lab_1/lab-1.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+4. Створюю нову вітку зі своїм іменем.
+5. Відкриваю редактор коду і пишу у ньому свою першу програму.
+6. Добавляю файл програми до репозиторію, також заповнюю документацію.
+7. За допомогою команди "git add ." добавляю всі файли в стан очікування черги надсилання.
+8. За допомогою команди "git commit -m "text" " пишу коментарі, щодо змін, які зробив.
+9. За допомогою команди "git push" відправляю все на хмару.
+10. Заходжу на GitHub і натискаю "pull request".
+
+#### Висновки. На даній лабораторній роботі я познайомився з основами мови програмування Python, також навчився
+#### надсилати програму до репозиторії на GitHub, щоб викладач міг ознайомитися з кодом.
```
Comment (tatomyr) [inline Taras_Vasylychyn/lab_1/lab-1.md]:
```suggestion
## Висновки. 

На даній лабораторній роботі я познайомився з основами мови програмування Python, також навчився надсилати програму до репозиторії на GitHub, щоб викладач міг ознайомитися з кодом. 
```

File: Taras_Vasylychyn/lab_1/lab-1.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+1. Створив аккаунт на GitHub.
+2. Отримав запрошення до репозиторію, підєднуюся до нього.
+3. Через термінал за допомогою команди "git clone /link/" клоную репозиторій до свого компютера.
+4. Створюю нову вітку зі своїм іменем.
+5. Відкриваю редактор коду і пишу у ньому свою першу програму.
+6. Добавляю файл програми до репозиторію, також заповнюю документацію.
+7. За допомогою команди "git add ." добавляю всі файли в стан очікування черги надсилання.
+8. За допомогою команди "git commit -m "text" " пишу коментарі, щодо змін, які зробив.
+9. За допомогою команди "git push" відправляю все на хмару.
+10. Заходжу на GitHub і натискаю "pull request".
```
Comment (tatomyr) [inline Taras_Vasylychyn/lab_1/lab-1.md]: Link to your code, repo...

**Review by tatomyr — CHANGES_REQUESTED** [review]
Please make changes to all documents according to my suggestions.

File: Taras_Vasylychyn/lab-5/lab-5.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+|----------------------------------------------|
+| Перевірив: Татомир Андрій                    |
+
+
+**Мета: навчитися оперувати класами та об'єктами.**
+
+
+Хід роботи
+
+Клас на Python - це сукупність об’єктів, ми можемо сприймати клас як план, ескіз чи прототип. Він містить усі деталі об’єкта.
```
Comment (tatomyr) [inline Taras_Vasylychyn/lab-5/lab-5.md:21]: Не зовсім!

File: Taras_Vasylychyn/lab-5/lab-5.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+У реальному прикладі тварина - це клас, оскільки у нас є різні види тварин у світі, і всі вони належать до класу під назвою тварини.
+```
+class MyClass: a = 10 b = 20 
+def add(): sum = a+b 
+print(sum)
```
Comment (pettrichor) [inline Taras_Vasylychyn/lab-5/lab-5.md]:
```suggestion
class MyClass: 
     a = 10 
     b = 20    
     def add(): 
     sum = a+b 
     print(sum)
```

File: Taras_Vasylychyn/lab-5/lab-5.md [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+Клас на Python - це сукупність об’єктів, ми можемо сприймати клас як план, ескіз чи прототип. Він містить усі деталі об’єкта.
+У реальному прикладі тварина - це клас, оскільки у нас є різні види тварин у світі, і всі вони належать до класу під назвою тварини.
+```
+class MyClass: 
+     a = 10 
+     b = 20    
+     def add(): 
+     sum = a+b 
+     print(sum)
+```
```
Comment (pettrichor) [inline Taras_Vasylychyn/lab-5/lab-5.md]:
```suggestion
class MyClass: 
     a = 10 
     b = 20    
     def add(): 
        sum = a+b 
        print(sum)
```
```

**Review by tatomyr — APPROVED** [review]

**Pull request merged**